am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -22,16 +22,15 @@
 @RegisterColumnMapper(PermissionSetMapper.class)
 public interface AccessManagementRepository {
 
-    @SqlUpdate("insert into access_management (resource_id, accessor_id, permissions, access_type, service_name, resource_type, resource_name, attribute, security_classification) "
-        + "values (:resourceId, :accessorId, :permissionsAsInt, :accessType, :serviceName, :resourceType, :resourceName, :attributeAsString, :securityClassification)"
-        + "on conflict on constraint access_management_unique do update set permissions = :permissionsAsInt"
-    )
+    @SqlUpdate("insert into access_management (resource_id, accessor_id, permissions, access_type, service_name, resource_type, resource_name, attribute, security_classification, relationship) "
```

Comment:
where we meant to drop security classification for explicit access records?

Reply:
Same as below